src/Factory/ClientFactory.php

```diff
@@ -50,43 +51,8 @@ public function create(array $options = [], array $subscribers = [])
         return $client;
     }
 
-    public function createNamed($alias)
```

Comment:
I was not sure what to do with this part here in the sense that these are public methods. Seeing as they're not documented and part of the implementation detail, I considered these methods to be non-API.